src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/components/step_index_pattern/__tests__/__snapshots__/step_index_pattern.test.js.snap

```diff
@@ -11,52 +11,17 @@ exports[`StepIndexPattern should disable the next step if the index pattern exis
     errors={Array []}
     goToNextStep={[Function]}
     isInputInvalid={false}
-    isNextStepDisabled={false}
+    isNextStepDisabled={true}
```

Comment:
This doesn't look right.

Because of the addition of extra Promise layer, we need to perform another `await` in the test.

The test should look like:
```
it('should disable the next step if the index pattern exists', async () => {
    const component = shallow(
      <StepIndexPattern
        allIndices={allIndices}
        isIncludingSystemIndices={false}
        esService={esService}
        savedObjectsClient={{
          find: () => ({ savedObjects: [
            { attributes: { title: 'k*' } }
          ] })
        }}
        goToNextStep={goToNextStep}
        initialQuery="k*"
      />
    );

    // Allow the componentWillMount code to execute
    // https://github.com/airbnb/enzyme/issues/450
    await component.update(); // Fire `componentWillMount()`
    await component.update(); // Force update the component post async actions
    await component.update(); // Ensure the reasonable wait finishes

    expect(component).toMatchSnapshot();
  });
```

Reply:
Updated this comment @cjcenizal

Reply:
For #2, the code is https://github.com/elastic/kibana/pull/16895/files#diff-ac3256ee9799125617e80b1b906d9d78R51

Reply:
Also, see #16968